transformer:

  ball:
    ball_topic: 'ball_in_image'
    # ballradius in meter
    ball_radius: 0.075

  lines:
    lines_topic: 'line_in_image'
    # if the line_in_image message is used to only to sent points and a point cloud should be published
    lines_relative: true
    pointcloud: true

  goals:
    goals_topic: 'goal_in_image'

  obstacles:
    obstacles_topic: 'obstacles_in_image'

  camera_info:
    camera_info_topic: '/camera/camera_info'